test/providers/tst_manifests/npm/package_json_deps_with_exhortignore_object/stack_expected_sbom.json

```diff
@@ -1,1870 +1,2533 @@
-				"pkg:npm/glob-parent@5.1.2",
-				"pkg:npm/is-binary-path@2.1.0",
-				"pkg:npm/is-glob@4.0.3",
-				"pkg:npm/normalize-path@3.0.0",
-				"pkg:npm/readdirp@3.6.0"
-			]
-		},
-		{
-			"ref": "pkg:npm/anymatch@3.1.3",
-			"dependsOn": [
-				"pkg:npm/normalize-path@3.0.0",
-				"pkg:npm/picomatch@2.3.1"
-			]
-		},
-		{
-			"ref": "pkg:npm/normalize-path@3.0.0",
-			"dependsOn": []
-		},
-		{
-			"ref": "pkg:npm/picomatch@2.3.1",
-			"dependsOn": []
-		},
-		{
-			"ref": "pkg:npm/braces@3.0.2",
-			"dependsOn": [
-				"pkg:npm/fill-range@7.0.1"
-			]
-		},
-		{
-			"ref": "pkg:npm/fill-range@7.0.1",
-			"dependsOn": [
-				"pkg:npm/to-regex-range@5.0.1"
-			]
-		},
-		{
-			"ref": "pkg:npm/to-regex-range@5.0.1",
-			"dependsOn": [
-				"pkg:npm/is-number@7.0.0"
-			]
-		},
-		{
-			"ref": "pkg:npm/is-number@7.0.0",
-			"dependsOn": []
-		},
-		{
-			"ref": "pkg:npm/fsevents@2.3.2",
-			"dependsOn": []
-		},
-		{
-			"ref": "pkg:npm/glob-parent@5.1.2",
-			"dependsOn": [
-				"pkg:npm/is-glob@4.0.3"
-			]
-		},
-		{
-			"ref": "pkg:npm/is-glob@4.0.3",
-			"dependsOn": [
-				"pkg:npm/is-extglob@2.1.1"
-			]
-		},
-		{
-			"ref": "pkg:npm/is-binary-path@2.1.0",
-			"dependsOn": [
-				"pkg:npm/binary-extensions@2.2.0"
-			]
-		},
-		{
-			"ref": "pkg:npm/binary-extensions@2.2.0",
-			"dependsOn": []
-		},
-		{
-			"ref": "pkg:npm/is-extglob@2.1.1",
-			"dependsOn": []
-		},
-		{
-			"ref": "pkg:npm/readdirp@3.6.0",
-			"dependsOn": [
-				"pkg:npm/picomatch@2.3.1"
-			]
-		},
-		{
-			"ref": "pkg:npm/debug@3.2.7",
-			"dependsOn": [
-				"pkg:npm/ms@2.1.3"
-			]
-		},
-		{
-			"ref": "pkg:npm/ignore-by-default@1.0.1",
-			"dependsOn": []
-		},
-		{
-			"ref": "pkg:npm/minimatch@3.1.2",
-			"dependsOn": [
-				"pkg:npm/brace-expansion@1.1.11"
-			]
-		},
-		{
-			"ref": "pkg:npm/brace-expansion@1.1.11",
-			"dependsOn": [
-				"pkg:npm/balanced-match@1.0.2",
-				"pkg:npm/concat-map@0.0.1"
-			]
-		},
-		{
-			"ref": "pkg:npm/balanced-match@1.0.2",
-			"dependsOn": []
-		},
-		{
-			"ref": "pkg:npm/concat-map@0.0.1",
-			"dependsOn": []
-		},
-		{
-			"ref": "pkg:npm/pstree.remy@1.1.8",
-			"dependsOn": []
-		},
-		{
-			"ref": "pkg:npm/simple-update-notifier@1.1.0",
-			"dependsOn": [
-				"pkg:npm/semver@7.0.0"
-			]
-		},
-		{
-			"ref": "pkg:npm/semver@7.0.0",
-			"dependsOn": []
-		},
-		{
-			"ref": "pkg:npm/supports-color@5.5.0",
-			"dependsOn": [
-				"pkg:npm/has-flag@3.0.0"
-			]
-		},
-		{
-			"ref": "pkg:npm/has-flag@3.0.0",
-			"dependsOn": []
-		},
-		{
-			"ref": "pkg:npm/touch@3.1.0",
-			"dependsOn": [
-				"pkg:npm/nopt@1.0.10"
-			]
-		},
-		{
-			"ref": "pkg:npm/nopt@1.0.10",
-			"dependsOn": [
-				"pkg:npm/abbrev@1.1.1"
-			]
-		},
-		{
-			"ref": "pkg:npm/abbrev@1.1.1",
-			"dependsOn": []
-		},
-		{
-			"ref": "pkg:npm/undefsafe@2.0.5",
-			"dependsOn": []
-		}
-	]
-}
+    "bomFormat": "CycloneDX",
+    "specVersion": "1.4",
+    "version": 1,
+    "metadata": {
+        "timestamp": "2023-09-20T14:27:17.520Z",
```

Comment:
@ruromero 
Just rename the timestamp back so tests will succeed.
Thanks.

Reply:
Oh, I missed it